src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -145,7 +146,7 @@ class BaseCreateCourseForm extends React.Component {
             type="date"
             component={RenderInputTextField}
             format={value => getDateString(value)}
-            normalize={value => moment.utc(value).toISOString()}
+            normalize={value => moment.utc(value).format(DATE_FORMAT)}
```

Comment:
Our API doesn't use ISO format? :(

Reply:
Not exactly. The standard for ISO is using milliseconds (SS.sss) and we do not use the milliseconds. This allows the fields to match up and pristine to be set. If things change in the future, should be an easy change to go to `toISOString`